datafusion/functions-array/src/array_has.rs

```diff
@@ -54,11 +54,17 @@ make_udf_function!(ArrayHasAny,
 );
 
 #[derive(Debug)]
-pub(super) struct ArrayHas {
```

Comment:
It seems that all the arrays function are public, do you plan to public the rest of them too?

Reply:
Yes, this should include all of functions from `functions` and `functions-array` crates at the time of authoring.